MovieGPT-api/src/main/java/lt/techin/group/project/model/User.java

```diff
@@ -18,9 +19,10 @@ public class User {
 
     private String username;
     private String email;
+    @JsonIgnore
     private String password;
```

Comment:
Lets have private String Name as well :)

Reply:
Decided to leave it as it is for now. In the future, we can implement new states.